Configuration/TCA/Overrides/tt_content.php

```diff
@@ -118,4 +118,4 @@
 if (!is_array($GLOBALS['TCA']['tt_content']['types']['mailform']['columnsOverrides']['bodytext'])) {
     $GLOBALS['TCA']['tt_content']['types']['mailform']['columnsOverrides']['bodytext'] = [];
 }
-$GLOBALS['TCA']['tt_content']['types']['mailform']['columnsOverrides']['bodytext']['config']['renderType'] = 'formwizard';
+$GLOBALS['TCA']['tt_content']['types']['mailform']['columnsOverrides']['bodytext']['config']['renderType'] = 'text';
```

Comment:
can you explain the reasoning behind this change a bit more?

Reply:
sure, the formwizard was shipped until v8 (was based on extjs), with v9 we do not have extjs anymore, so the wizard is gone.

A valid workaround is to simply show the textarea with the configuration, so that it can still be adjusted. If needed you may now configure it in v7 or v8 and copy the config to that field

Reply:
Perfect. Thanks. Should we also modify the label then to explain it?

Reply:
we can, just wanted to ensure, that it is atleast possible to edit the form again and see no exception.

what do you expect as label?